<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartWindow</class>
 <widget class="QMainWindow" name="PartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWChartView" name="chartView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBoundary"/>
   <addaction name="actionHoleNew"/>
   <addaction name="actionCancelSelect"/>
   <addaction name="actionReSelect"/>
   <addaction name="actionOK"/>
   <addaction name="actionSubmittedModel"/>
   <addaction name="actionModelControlData"/>
   <addaction name="actionMesh"/>
   <addaction name="actionClearMesh"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomReset"/>
  </widget>
  <action name="actionBoundary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/Boundary.png</normaloff>:/images/images/Boundary.png</iconset>
   </property>
   <property name="text">
    <string>定义边界</string>
   </property>
   <property name="toolTip">
    <string>定义边界</string>
   </property>
  </action>
  <action name="actionHoleNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/hole.png</normaloff>:/images/images/hole.png</iconset>
   </property>
   <property name="text">
    <string>新建孔洞</string>
   </property>
   <property name="toolTip">
    <string>新建孔洞</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/418.bmp</normaloff>:/icons/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/416.bmp</normaloff>:/icons/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionZoomReset">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/414.bmp</normaloff>:/icons/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>原始大小</string>
   </property>
   <property name="toolTip">
    <string>原始大小</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/exit.bmp</normaloff>:/icons/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionOK">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/322.bmp</normaloff>:/icons/icons/322.bmp</iconset>
   </property>
   <property name="text">
    <string>确认导入</string>
   </property>
   <property name="toolTip">
    <string>确认导入</string>
   </property>
  </action>
  <action name="actionCancelSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/remove.bmp</normaloff>:/icons/icons/remove.bmp</iconset>
   </property>
   <property name="text">
    <string>取消选择</string>
   </property>
   <property name="toolTip">
    <string>取消选择</string>
   </property>
  </action>
  <action name="actionReSelect">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/preview1.bmp</normaloff>:/icons/icons/preview1.bmp</iconset>
   </property>
   <property name="text">
    <string>清空选择</string>
   </property>
   <property name="toolTip">
    <string>清空选择</string>
   </property>
  </action>
  <action name="actionSubmittedModel">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/124.bmp</normaloff>:/icons/icons/124.bmp</iconset>
   </property>
   <property name="text">
    <string>查看已建模型</string>
   </property>
   <property name="toolTip">
    <string>查看已建模型</string>
   </property>
  </action>
  <action name="actionMesh">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/130.bmp</normaloff>:/icons/icons/130.bmp</iconset>
   </property>
   <property name="text">
    <string>划分网格</string>
   </property>
   <property name="toolTip">
    <string>划分网格</string>
   </property>
  </action>
  <action name="actionClearMesh">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/107.bmp</normaloff>:/icons/icons/107.bmp</iconset>
   </property>
   <property name="text">
    <string>清除网格</string>
   </property>
   <property name="toolTip">
    <string>清除网格</string>
   </property>
  </action>
  <action name="actionModelControlData">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/126.bmp</normaloff>:/icons/icons/126.bmp</iconset>
   </property>
   <property name="text">
    <string>生成模型数据</string>
   </property>
   <property name="toolTip">
    <string>生成模型数据</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qwchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>PartWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
